lms/djangoapps/grades/tests/test_tasks.py

```diff
@@ -137,28 +141,28 @@ def test_block_structure_created_only_once(self):
             self.assertEquals(mock_block_structure_create.call_count, 1)
 
     @ddt.data(
-        (ModuleStoreEnum.Type.mongo, 1, 23),
-        (ModuleStoreEnum.Type.split, 3, 22),
+        (ModuleStoreEnum.Type.mongo, 1, 24, True),
+        (ModuleStoreEnum.Type.mongo, 1, 21, False),
+        (ModuleStoreEnum.Type.split, 3, 23, True),
+        (ModuleStoreEnum.Type.split, 3, 20, False),
     )
     @ddt.unpack
-    def test_query_counts(self, default_store, num_mongo_calls, num_sql_calls):
+    def test_query_counts(self, default_store, num_mongo_calls, num_sql_calls, use_bulk_ops):
```

Comment:
Nit: Can we rename this parameter to 'create_multiple_subsections` or the like?  It's unclear what "bulk ops" refers to in this context.